streisand

```diff
@@ -169,13 +169,21 @@ Please enter the word 'streisand' to continue: "
   else
     SSH_USER=${ANSIBLE_SSH_USER}
   fi
+  
+  # If ANSIBLE_SSH_PORT is empty, default to root
+  if [ -z "${ANSIBLE_SSH_PORT}" ]; then
```

Comment:
this is a bit pedantic, but this would avoid the `else`  (can be done in the ANSIBLE_SSH_USER lines above too):
```bash
SSH_PORT='22'
if [ -n "${ANISIBLE_PORT}" ]; then
    SSH_PORT=${ANISIBLE_PORT}
fi
```

it looks like Ansible has [deprecated](https://docs.ansible.com/ansible/2.3/intro_inventory.html#list-of-behavioral-inventory-parameters) variables like `ANSIBLE_SSH_PORT` in favour of `ansible_port`  (as seen in line 186).. I think we should follow and call our vars `ANSIBLE_USER` and `ANSIBLE_PORT`. 

In the related issues you linked, I noticed folks were asking for this to be documented. I think this PR would be a good opportunity to add some documentation for these custom ANSIBLE_USER and ANSIBLE_PORT settings.